### ISSUE TITLE
better error message for small container for table

### ISSUE THREAD

**Issue by Alan-M-Thomaz** [prawnpdf/prawn-table#111]
`Table's width was set too small to contain its contents (min width 540.0, requested 523.28)`
when called for `pdf.table(table_content, header: true, column_widths: [100, 100,  340])`
that is a bit confuse, since i "requested" 540 width in column_widths

the problem was that the container(page, bounding_box, or whatever the table is in) it self was 523.28, take me a while to figure that the problem, would be better some message like `Container is to small to contain the table(requested width: 540, container width: 523.28`

**Comment by pointlessone** [comment]
I agree the message is confusing. Would you have some time to contribute a fix for this?
